modules/customize-certificates-replace-default-router.adoc

```diff
@@ -47,6 +37,13 @@ $ oc patch proxy/cluster \
      --type=merge \
      --patch='{"spec":{"trustedCA":{"name":"custom-ca"}}}'
 ----
++
+[IMPORTANT]
+====
+After you run the `oc patch` command, the Machine Config Operator (MCO) reboots all nodes in your cluster so that each node receives the updated certificates.
```

Comment:
A note does exist in https://docs.redhat.com/en/documentation/openshift_container_platform/4.15/html/networking/enable-cluster-wide-proxy#nw-proxy-configure-object_config-cluster-wide-proxy that details:

> Enabling the cluster-wide proxy causes the Machine Config Operator (MCO) to trigger node reboot.

Maybe this note is not well placed?

Reply:
Updating the ingress certificate on each node does not cause the MCO to reboot nodes. To update the ingress certificate, create a secret and then use the `oc patch` command to apply the new secret to the `ingresscontroller.operator` custom resource (CR).